Policies/github-community-guidelines.md

```diff
@@ -79,7 +79,16 @@ We are committed to maintaining a community where users are free to express them
    You may not post content that presents a distorted view of reality, whether it is inaccurate or false (misinformation) or is intentionally deceptive (disinformation) where such content is likely to result in harm to the public or to interfere with fair and equal opportunities for all to participate in public life. For example, we do not allow content that may put the well-being of groups of people at risk or limit their ability to take part in a free and open society. We encourage active participation in the expression of ideas, perspectives, and experiences and may not be in a position to dispute personal accounts or observations. We generally allow parody and satire that is in line with our Acceptable Use Polices, and we consider context to be important in how information is received and understood; therefore, it may be appropriate to clarify your intentions via disclaimers or other means, as well as the source(s) of your information.
 
 - #### Active malware or exploits
-   Being part of a community includes not taking advantage of other members of the community. We do not allow anyone to use our platform for exploit delivery, such as using GitHub as a means to deliver malicious executables, or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Note, however, that we do not prohibit the posting of source code which could be used to develop malware or exploits, as the publication and distribution of such source code has educational value and provides a net benefit to the security community.
+   Being part of a community includes not taking advantage of other members of the community. We do not allow anyone to use our platform in support of active attacks that cause harm, such as using GitHub as a means to deliver malicious executables, or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers.
+
+  Note, however, that GitHub supports the posting of content which is used for research into vulnerabilities, malware or exploits, as the publication and distribution of such content has educational value and provides a net benefit to the security community. We ask that repository owners take the following steps when posting potentially harmful content for the purposes of security research:
+
+  * Clearly identify and describe any potentially harmful content in a disclaimer in the project’s README.md file.
```

Comment:
> potentially harmful content

This clause would likely be better received by the security community if the same standards were applied to all code. There's plenty of sys admin scripts that can completely ruin your systems... is there a similar restriction placed on them?  The placement of the clause confuses me. Is it for everyone, or just code that could be problematic?

Reply:
> There's plenty of sys admin scripts that can completely ruin your systems...

Agreed - what the suggested change does is pushes responsibility onto the commiter whilst providing no clear definition of "potentially harmful".

`rm` is potentially harmful, but presumably wouldn't need to declare it.

There are Nessus plugins that are potentially harmful if run against a production box, but they're not intended to do harm. Would they need to put a disclaimer in?